ccpp/suites/suite_FV3_GFS_v16_coupled_p8_sfcocn.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_GFS_v16_coupled_p8_sfcocn" version="1">
```

Comment:
Do we still need to add this new suite file for GFS_v16_coupled_p8?

Reply:
@junwang-noaa Currently, the suite file in CMEPS side uses sfc_ocean. If you want to performs side by side run to check fluxes computed with FV3/CCPP vs. CMEPS/CCPP you need this suite file (or you could edit existing file and replace the NSST with sfc_ocean). So, it is not must but I think it would be nice to have it for side-by-side comparison. If you want me to remove, I could do it.

Reply:
Can we do that for GFS_v17_coupled_p8?

Reply:
@junwang-noaa we have also for GFS_v17_coupled_p8. It is in the suite/ folder. I also updated the model and all the sub-components. Once I finish the testing, I'll push them to the branches and update you.